config/config.go

```diff
@@ -984,6 +984,7 @@ type StateSyncConfig struct {
 	DiscoveryTime       time.Duration `mapstructure:"discovery_time"`
 	ChunkRequestTimeout time.Duration `mapstructure:"chunk_request_timeout"`
 	ChunkFetchers       int32         `mapstructure:"chunk_fetchers"`
+	TargetHeight 	  	int64         `mapstructure:"target_height"`
```

Comment:
Could you add a validation rule in `ValidateBasic` to check that `TargetHeight` is `>= 0`?